### PR TITLE
Allow use plugin array & add plural web3js helpers

### PR DESCRIPTION
**Pull request by mPaella**
- allow plugins to be passed as an array
- add plural web3js-adapter functions

### CODE REVIEW

File: docs/web3js-adapters.md [hunk 1]
```diff
@@ -17,26 +17,39 @@ Then, you will have access to a bunch of helper methods to convert to and from W
...
 toWeb3JsMessage(myUmiTransactionMessage);
+toWeb3JsMessages([myUmiTransactionMessage, myOtherUmiTransactionMessage]);
 toWeb3JsMessageFromInput(myUmiTransactionInput);
+toWeb3JsMessagesFromInputs([myUmiTransactionInput, myOtherUmiTransactionInput]);
 
 // For instructions.
 fromWeb3JsInstruction(myWeb3JsInstruction);
+fromWeb3JsInstructions([myWeb3JsInstruction, myOtherWeb3JsInstruction]);
 toWeb3JsInstruction(myUmiInstruction);
+toWeb3JsInstructions([myUmiInstruction, myOtherUmiInstruction]);
```
Comment (mPaella) [inline docs/web3js-adapters.md:52]: this is kinda ugly, so happy to do in a different way if you'd like, for ex just specifying in text that these functions all have plural equivalents

**Comment by lorisleiva** [comment]
Hey, thanks for the PR! I'm really not a fan of the plural web3js adapters though. We're only saving the end-user to do `array.map(someFunction)` which I don't think requires a wrapper.

Regarding accepting an array of plugin do you have a use case for this? Because currently we encourage people to do this by creating bundles of plugins like so:

```ts
const myBundle = (): UmiPlugin => {
  install(umi: Umi) {
    umi.use(pluginA())
      .use(pluginB())
      .use(pluginC())
      .use(pluginD());
  }
}
```

Meaning the end-user can install all these plugins like so:

```ts
umi.use(myBundle());
```

Which I think is more elegant than:

```ts
umi.use([pluginA(), pluginB(), pluginC(), pluginD()]);
```

**Comment by mPaella** [comment]
> Hey, thanks for the PR! I'm really not a fan of the plural web3js adapters though. We're only saving the end-user to do `array.map(someFunction)` which I don't think requires a wrapper.

i think you could make the same argument in the other way as well, that since its a single line the lib should just have it, but i understand...

stretch, but would you accept this?

``` typescript
export function fromWeb3JsPublicKey(publicKey: Web3JsPublicKey): PublicKey;
export function fromWeb3JsPublicKey(
  ...publicKeys: Web3JsPublicKey[]
): PublicKey[];
export function fromWeb3JsPublicKey(
  ...publicKeys: Web3JsPublicKey[]
): PublicKey | PublicKey[] {
  if (publicKeys.length === 1) {
    return publicKeys[0].toBase58() as PublicKey;
  }
  return publicKeys.map((publicKey) => publicKey.toBase58() as PublicKey);
}

// usage
const pk1 = fromWeb3JsPublicKey(new Web3JsPublicKey('pk1'));

const [pk1, pk2] = fromWeb3JsPublicKey(
  new Web3JsPublicKey('pk1'),
  new Web3JsPublicKey('pk1')
);
```

**Comment by lorisleiva** [comment]
I'd rather leave things as-is for now. There's gonna be a few changes in that department when I update Umi to use the new Web3.js and I think this is an unnecessary refactoring at this time.

**Pull request closed without merging**